web/assets/todays-date.js

```diff
@@ -0,0 +1,12 @@
+const todaysDate = () => {
+    let selectTodayLink = document.querySelector('[data-module="select-todays-date"]');
+    let datePicker = document.querySelector('.date-picker');
```

Comment:
I'm not very good with JS so i'm not exactly sure of when/where I should be using `data-module=...` or just a custom class like `date-picker` ?